src/shaders/Sky.tsx

```diff
@@ -1,11 +1,16 @@
 import React, { forwardRef, useMemo } from 'react'
-import { ReactThreeFiber, extend } from 'react-three-fiber'
+import { ReactThreeFiber } from 'react-three-fiber'
 import { Sky as SkyImpl } from 'three/examples/jsm/objects/Sky'
 import { Vector3 } from 'three'
 
 export type Sky = {
   distance?: number
-  sunPosition?: ReactThreeFiber.Vector3
+  // sunPosition?: ReactThreeFiber.Vector3
```

Comment:
why splitting it?

Reply:
I split it based on your message in the Discord channel: 

"to be honest I would separate the three angles in three props"

Let me know if you envisioned something else; this implementation didn't make much sense to me since I couldn't see a scenario in which an end-user would want to declare the three angles separately.

Reply:
Yeah I meant three different angles, not three different positions tbh, otherwise it's better using a `ReactThreeFiber.Vector3`.

Either way it may be a bad idea to keep both angles and position, which should have priority?

Reply:
As someone who recently tried it for the first time it would make much more sense to me to just expose `inclination` and `azimuth` to the end user and to handle the angle definitions internally without exposing that logic. 

As long as it mimics how the angles are defined in the THREE example, that should be enough to let the end-user do what they want while retaining parity with the source example.

Reply:
Ok so we could remove sunPosition entirely 😛

Reply:
Awesome, I'll have that changed later today